tests/Integration/inc/Addon/GoogleTracking/GoogleAnalytics/replaceUrl.php

```diff
@@ -0,0 +1,39 @@
+<?php
+namespace WP_Rocket\Tests\Integration\inc\Addon\GoogleTracking\GoogleAnalytics;
+
+use WP_Rocket\Tests\Integration\FilesystemTestCase;
+use Brain\Monkey\Functions;
+
+/**
+ * @covers \WP_Rocket\Addon\GoogleTracking\GoogleAnalytics::replace_url
+ * @group  Addon
+ * @group  GoogleTracking
+ */
+class Test_ReplaceUrl extends FilesystemTestCase {
+	protected $path_to_test_data = '/inc/Addon/GoogleTracking/GoogleAnalytics/replaceUrl.php';
+
+	public function tearDown() {
+		parent::tearDown();
+		remove_filter('pre_get_rocket_option_google_analytics_cache', '__return_true');
```

Comment:
```suggestion
		remove_filter( 'pre_get_rocket_option_google_analytics_cache', [ $this, 'return_true' ] );
```

That way, if `__return_true` is ever used on this filter somewhere else, we do not remove it completely during testing.